### PR TITLE
Changed the way the S3 transformation info is given; Added tests.

### PR DESCRIPTION
**Pull request by timtreis**
Changed the way the info that is given when transforming a non-survfit object using visR::tidyme() from base::writeLines() to message.

Then wrote these tests:

#' T1.  The function properly converts "messy output" into tidy tibbles
#' T1.1 The `visR::tidyme(stats::lm())` transformation doesn't change the values
#' T1.2 The `visR::tidyme(survival::survfit())` transformation doesn't change the values
#' T2.  Warnings and errors are thrown when neccecary
#' T2.1 No error when a `stats::lm()` model is transformed
#' T2.2 No error when a `survival::survfit()` model is transformed
#' T2.3 Message when a non-survfit object is transformed

If accepted, branch can be merged and deleted.

### CODE REVIEW

File: R/tidyme.R [hunk 1]
```diff
@@ -40,7 +40,7 @@ tidyme <- function(x, ...){
 
 tidyme.default <- function(x, ...){
   #library(broom)
-  base::writeLines("tidyme S3 default method (broom::tidy) used.")
+  message("tidyme S3 default method (broom::tidy) used.")
```
Comment (SHAESEN2) [inline R/tidyme.R:43]: include prefix base::

File: tests/testthat/test-tidyme.R [hunk 2]
```diff
@@ -0,0 +1,86 @@
+#' @title tidyme
+#' @section Last updated by:
+#' Tim Treis
+#' @section Last update date:
+#' 19-MAR-2021
+
+# Specifications ---------------------------------------------------------------
+
+#' T1.  The function properly converts "messy output" into tidy tibbles
```
Comment (SHAESEN2) [inline tests/testthat/test-tidyme.R:9]: Properly is vague.
T1. The function tidies up an associated object
T1.1 The default method returns a tibble, tidies with broom
T1.2 The default method does not affect the content of the object
T1.3 The S3 method, associated with a survival object, returns a ...
T2 The S3  method, associated with a survival object, transforms the survival object into a data frame
T2.1 Which information is retained and which is not??
T2.X .....

File: tests/testthat/test-tidyme.R [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+#' T2.2 No error when a `survival::survfit()` model is transformed
+#' T2.3 Message when a non-survfit object is transformed
+
+# Requirement T1 ---------------------------------------------------------------
+
+context("tidyme - The function properly converts \"messy output\" into tidy tibbles")
+
+testthat::test_that("T1.1 The `visR::tidyme(stats::lm())` transformation doesn't change the values",{
+  
+  lm_object <- stats::lm(data = adtte, TRTDUR ~ AVAL)
```
Comment (SHAESEN2) [inline tests/testthat/test-tidyme.R:23]: I would compare tidy::broom(lm) with tidyme(lm). Is much easier, but on the other hand your approach is much more robust.

File: tests/testthat/test-tidyme.R [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  
+  testthat::expect_equal(c(0.06, 0.09, 0.46, 2.07), round(cumhaz, 2))
+  testthat::expect_equal(round(cumhaz, 2), round(cumhaz_tidied, 2))
+  
+})
+
+
+# Requirement T2 ---------------------------------------------------------------
+
+context("tidyme - T2. Warnings and errors are thrown when neccecary")
```
Comment (SHAESEN2) [inline tests/testthat/test-tidyme.R:60]: very vague requirement. Define 'when'

**Review by SHAESEN2 — CHANGES_REQUESTED** [review]

File: tests/testthat/test-tidyme.R (hunk 3 above)
Comment (timtreis) [reply tests/testthat/test-tidyme.R:23]: Then I'd just stick with the more robust one?

**Comment by timtreis** [comment]
Info: Some changes need to be reverted to make a 2.2 functional again. Will update later.

**Pull request closed without merging**